Week-2/Assignment 2.1/Style.css

```diff
@@ -0,0 +1,24 @@
+.row {
```

Comment:
file name to be consistent across all files

Reply:
Sure.